build/run.js

```diff
@@ -171,7 +171,7 @@ commands.build.rust = async function(argv) {
 
         console.log('Checking the resulting WASM size.')
         let stats = fss.statSync(paths.dist.wasm.mainOptGz)
-        let limit = 4.29
+        let limit = 4.37
```

Comment:
This is blocker to me. This is HUGE change. Where it comes from??? Remember, that parser is NOT included in this file size, so what Rust code changes were here that caused this increase?